src/jobs/run.yml

```diff
@@ -48,6 +52,8 @@ parameters:
 executor: <<parameters.executor>>
 
 steps:
+  - restore_cache:
+      key: rainforest-run-{{ .Revision }}-
```

Comment:
You may wish to add both cache keys here to ensure the cache is pulled specifically in the order you choose. You may want to place "rainforest-run-{{ .Revision }}-{{ .BuildNum }}" above this.

https://circleci.com/docs/2.0/caching/#full-example-of-saving-and-restoring-cache